03/app.js

```diff
@@ -7,12 +7,68 @@ const c = randomNumber(min, max);
 
 console.log(a, b, c);
 
+function randomNumber(min, max) {
+    return Math.round((Math.random() * (max - min)) + min);
+}
 
+function GetSum(par1, par2, par3) {
```

Comment:
Funkcje powinniśmy napisać z małej litery, o ile to nie jest konstruktor (ta funkcja nim nie jest)